src/Sia.Gateway/Sia.Gateway.csproj

```diff
@@ -37,7 +37,6 @@
     <ProjectReference Include="..\Sia.Data.Incident\Sia.Data.Incidents.csproj" />
   </ItemGroup>
   <ItemGroup>
-    <Folder Include="Controllers\Playbook\" />
```

Comment:
I have no idea why this line went away, but it doesn't seem to have hurt anything.